packit/config/package_config.py

```diff
@@ -194,7 +194,19 @@ def load_packit_yaml(
         return safe_load(raw_text) or {}
     except YAMLError as ex:
         logger.error(f"Cannot load package config {config_file_path}.")
-        raise PackitConfigException(f"Cannot load package config: {ex!r}.")
+        if hasattr(ex, "problem_mark"):
+            msg = (
+                "  parser says\n"
+                + str(ex.problem_mark)
+                + "\n  "
+                + str(ex.problem)  # type: ignore
+            )  # type: ignore
```

Comment:
```python
msg = f"  parser says\n{ex.problem_mark}\n  {ex.problem}"  # type: ignore
```
works OK (pre-commit doesn't complain) for me, but I don't mind having it multi-line